WeakAuras/Transmission.lua

```diff
@@ -762,15 +809,6 @@ function WeakAuras.ShowDisplayTooltip(data, children, icon, icons, import, compr
       else
         tinsert(tooltip, {1, L["No Children:"], 1, 1, 1});
       end
-    elseif(data.controlledChildren) then
```

Comment:
Can you explain why you removed this code? It seems used to me.

Reply:
I couldn’t see a path to make this code execute, and it’s otherwise a duplication of the branch just above it.

Reply:
Take a close look at all callers of ShowDisplayTooltip, some call it with data but no children (and compressed == nil)